### PR TITLE
add boba.js@1.0.3 w/ npm auto-update

### LINKED ISSUE #8891: [Request] Add boba.js
**Library name:** boba.js
**Git repository url:** https://github.com/space150/boba.js
**npm package url(optional):**  https://www.npmjs.com/package/boba.js
**License(s):** MIT
**Official homepage:**http://boba.space150.com/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by emmy41124**
close #8891, cc @athaeryn, cc #9414

Pull request for issue: #8891 
Related issue(s): #9414

@nolsherry please help me review this PR, thank you
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/space150/boba.js
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/boba.js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 946
   - Count of watchers: 48
   - Count of forks: 34
 * NPM download stats (optional):
   - Downloads in the last day: 6
   - Downloads in the last week: 13
   - Downloads in the last month: 54

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/boba.js/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "name": "Mike Anderson",
+    "url": "http://github.com/athaeryn"
+  },
+  "license": "MIT",
+  "npmName": "boba.js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "boba.js*"
```
Comment (nolsherry) [inline ajax/libs/boba.js/package.json]: Please modify `boba.js` to `boba*` in the filemap, thanks!

File: ajax/libs/boba.js/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "git",
+    "url": "https://github.com/space150/boba.js"
+  },
+  "keywords": [
+    "analytics",
+    "tracking"
+  ],
+  "author": {
+    "name": "Mike Anderson",
+    "url": "http://github.com/athaeryn"
```
Comment (nolsherry) [inline ajax/libs/boba.js/package.json]: I think `https://github.com/athaeryn` is batter.

**Comment by nolsherry** [comment]
@emmy41124 please maintain a commit message, thanks!
You can use `git commit --amend` to modify the latest commit message.

**Comment by nolsherry** [comment]
LGTM, ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**